### PR TITLE
process documentation

### PR DESCRIPTION
**Pull request by kflemin**
adding process details for testing via CLI and example project development

### CODE REVIEW

File: developer_resources/developer_resources.md [hunk 1]
```diff
@@ -38,4 +38,24 @@ Follow these steps to generate rdocs for now and existing SDK gems:
 
 Rdoc options are controlled with this [config file](https://github.com/urbanopt/urbanopt-scenario-gem/blob/develop/.rdoc_options).
 
+## Developing with the CLI
+
+When developing locally on URBANopt core gems and testing new functionality and dependencies via the CLI, it may be necessary to install local versions of core gems.  
+
+For local development, you will want to set the environment variable *FAVOR_LOCAL_GEMS* to 1. This enables local copies of gems in the Gemfile.  There are 2 Gemfiles of interest:
```
Comment (vtnate) [inline developer_resources/developer_resources.md]: Can we add something like:

Remember that the only "falsey" things in Ruby are `nil` and `false`. Setting `FAVOR_LOCAL_GEMS==0` is still "truthy."

Comment (vtnate) [reply developer_resources/developer_resources.md]: If editing a gem that is already built, users will need to un-comment the appropriate block in the example project Gemfile.

File: developer_resources/developer_resources.md [hunk 2]
```diff
@@ -38,4 +38,24 @@ Follow these steps to generate rdocs for now and existing SDK gems:
...
+
+Follow these steps when testing local gems via the CLI:
+
+1. ```bundle install``` (to install the local gems that were just enabled in the Gemfile)
+1. ```bundle exec rake install``` (to install the CLI in a place that has access to those local gems)
+1. ```bundle exec uo ...``` (to execute the CLI calls)
+
+## Example Project Development
+
+1. Develop new functionality in the example project repo.
```
Comment (vtnate) [inline developer_resources/developer_resources.md]: Do we want to call this a repo or just a directory/folder?

Comment (kflemin) [reply developer_resources/developer_resources.md]: directory!

**Comment by kflemin** [comment]
@vtnate I tweaked the language.

**Pull request merged**